common/src/service/HttpClientHelper.cs

```diff
@@ -919,5 +894,37 @@ public void Dispose()
             Task<Exception> exception = mapToExceptionFunc(response);
             return await exception.ConfigureAwait(false);
         }
+
+        internal static HttpClient CreateDefaultClient(IWebProxy webProxy, Uri baseUri, int connectionLeaseTimeoutMilliseconds)
+        {
+
+#pragma warning disable CA2000 // Dispose objects before losing scope
+#if NETCOREAPP
+            SocketsHttpHandler httpMessageHandler = new SocketsHttpHandler();
+#else
+            HttpClientHandler httpMessageHandler = new HttpClientHandler();
+#endif
+#pragma warning restore CA2000 // Dispose objects before losing scope
+
+#if !NET451 && !NETCOREAPP
+            httpMessageHandler.SslProtocols = TlsVersions.Instance.Preferred;
+            httpMessageHandler.CheckCertificateRevocationList = TlsVersions.Instance.CertificateRevocationCheck;
+#endif
+
```

Comment:
I'm not sure if it is the diff here or if the indentation is really off for the lines below.

Reply:
Scratch that. It was fine when I did view file. Just looked weird here due to the #if directives.

Reply:
I probably goofed the spacing here. I'll fix it